api/v1alpha1/eventing_types.go

```diff
@@ -65,7 +65,7 @@ const (
 	ConditionReasonEventMeshSubManagerReady      ConditionReason = "EventMeshSubscriptionManagerReady"
 	ConditionReasonEventMeshSubManagerFailed     ConditionReason = "EventMeshSubscriptionManagerFailed"
 	ConditionReasonEventMeshSubManagerStopFailed ConditionReason = "EventMeshSubscriptionManagerStopFailed"
-	ConditionReasonEventMeshSubManagerStop       ConditionReason = "Stopped"
+	ConditionReasonEventMeshSubManagerStopped    ConditionReason = "Stopped"
```

Comment:
The ConditionReason for all other cases is `EventMeshSubscriptionManager<something>`. For the sake of consistency, can we have `EventMeshSubscriptionManagerStopped` here?

Otherwise, if none of them need to be EventMesh specific, can the other reasons have that part removed, so it can be reused for NATS?

Reply:
Reason stopped is actually common for NATS and EventMesh. Thus, changed to general name.